app/components/common/paperItem/authors.tsx

```diff
@@ -6,6 +6,7 @@ import { withStyles } from "../../../helpers/withStylesHelper";
 import { trackEvent } from "../../../helpers/handleGA";
 import GlobalDialogManager from "../../../helpers/globalDialogManager";
 import { Paper } from "../../../model/paper";
+import * as classNames from "classnames";
```

Comment:
Make import sentence that importing the external library be above the local module importing sentences.